packages/@rnw-scripts/doxysaurus/src/docusaurus.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+//
+// Copy generated documentation files to Docusaurus location.
+//
+
+import path from 'path';
+import {log} from './logger';
+import {promises as fs} from 'fs';
+
+export async function copyDocusaurusFiles(files: string[]) {
+  const docsPath = process.env.DOCUSAURUS_DOCS;
+  if (!docsPath) {
+    log(`[Not found] environment var {DOCUSAURUS_DOCS} to copy files`);
```

Comment:
Should this fail? I would forget to sent the env var, and a simple log would not catch my attention..

Reply:
at least a bit more bombastic like bright red would get my attenion :)

Reply:
It is intentionally optional. What if developer does not want to copy generated files to a docs repo? I hope in future we can move the doc site to the main repo and the need for the env variable will disappear.
Though I see your point and will convert it to a warning.

Reply:
The use of the `DOCUSAURUS_DOCS` environment variable is replaced with the new CLI `output` parameter in commit https://github.com/microsoft/react-native-windows/pull/7044/commits/9d9217ebec01386c2c58eec42c8bed6c73f76218